security/README.md

```diff
@@ -55,3 +38,62 @@ PGP and GnuPG
 * Don't share your private key with anyone, including services like Keybase.
 * Keep at least one backup of your private key and revocation certificate in a
   secure location, such as a thumb drive.
+
+## Physical Security
+
+The following guidelines apply to how we physically secure
+our laptops and mobile devices that may contain customer or user data.
+
+* Don't leave your devices unattended in an unsecured area.
+* Install a device tracking and remote data wipe tool such as [Prey].
+
+[Prey]: https://www.preyproject.com/
+
+## Application Security
+
+The following guidelines apply to how we develop
+software on behalf of ourselves and clients.
+
+### Transmitting Information
+
+* Don't accept passwords or session tokens over HTTP.
+* Use HTTPS for all web traffic.
+* Use HTTPS in the beginning; it's harder to introduce later.
+* Use HTTPS redirects for HTTP traffic.
+* Use [HSTS](http://tools.ietf.org/html/rfc6797) headers to enforce HTTPS
+  traffic.
+* Use secure cookies.
+* Avoid protocol-relative URLs.
+
+### Storing Information
+
+* Don't log passwords.
+* Don't store passwords in plain text.
+* Don't hash passwords using a reversible cipher.
+* Don't hash passwords using a broken cipher, such as MD5 or SHA1.
+
+## Handling Vulnerabilities
+
+The following guidelines apply to how we handle security incidents.
+
+### Reporting
+
+When someone finds a possible security issue in our software,
+we encourage them to report it to our <security@thoughtbot.com> email address.
+This email address is an alias for <everyone@thoughtbot.com>.
```

Comment:
Is putting this here going to result in that alias getting spammed? Is it enough to write the address out in a slightly obfuscated way and then to say, "it reaches everyone at thoughtbot directly"?